### PR TITLE
🏃‍♂️ Update build status badge on readme to use github actions

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashisham@vmware.com>

Fixing the badge on the readme now, buying time for travis removal.

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Push
+name: CI
```
Comment (nrb) [inline .github/workflows/push.yml:1]: Let's make this more even more descriptive, maybe "Container Image Push"?

Comment (ashish-amarnath) [reply .github/workflows/push.yml:1]: This is the text that shows up on the badge. It currently says "Push". I though "CI" would be a better name for the badge.

**Comment by skriss** [comment]
travis now appears to be failing, can we just remove it here?

**Comment by ashish-amarnath** [comment]
Going to open different PR to include travis removal.

**Pull request closed without merging**